frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/create-vm-wizard.tsx

```diff
@@ -406,6 +411,8 @@ const CreateVMWizard = connect(wizardStateToProps, wizardDispatchToProps, null,
     }),
 })(CreateVMWizardComponent);
 
+const imagesNamespace = isUpstream() ? KUBEVIRT_OS_IMAGES_NS : OPENSHIFT_OS_IMAGES_NS;
```

Comment:
suggestion: this already exists as `getOSImagesNS` in src/utils/common.ts

Reply:
done :)